core-services/prow/02_config/_config.yaml

```diff
@@ -504,9 +508,13 @@ plank:
       by [linking to](https://github.com/kubernetes/community/blob/master/contributors/devel/flaky-tests.md#filing-issues-for-flaky-tests)
       an [open issue](https://github.com/{{.Spec.Refs.Org}}/{{.Spec.Refs.Repo}}/issues?q=is:issue+is:open)
       when you hit one in your PR.'
-    integr8ly/integreatly-operator: |-
-      ### Report flaky E2E tests and PROW issues in [INTLY-7197](https://issues.redhat.com/browse/INTLY-7197) epic. Detailed instructions can be found in the description of said epic.
-      [Full PR test history](https://prow.svc.ci.openshift.org/pr-history?org={{.Spec.Refs.Org}}&repo={{.Spec.Refs.Repo}}&pr={{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}). [Your PR dashboard](https://prow.svc.ci.openshift.org/pr?query=is:pr+state:open+author:{{with index .Spec.Refs.Pulls 0}}{{.Author}}{{end}}).
+    integr8ly/integreatly-operator: '### Report flaky E2E tests and PROW issues in
```

Comment:
This delta came from python oyaml canonicalization. It's the same content, just using a different quoting mechanism. I can undo this piece if it bugs ya.

Reply:
[Later] Well, I guess it bugs CI, so...